hal/inc/exflash_hal.h

```diff
@@ -55,6 +56,7 @@ int hal_exflash_read(uintptr_t addr, uint8_t* data_buf, size_t data_size);
 int hal_exflash_copy_sector(uintptr_t src_addr, size_t dest_addr, size_t data_size);
 int hal_exflash_lock(void);
 int hal_exflash_unlock(void);
+int hal_exflash_otp_size(void);
```

Comment:
This should be done within `hal_exflash_special_command()` or some other generic method taking `hal_exflash_special_sector_t` as an argument. No need to introduce OTP-specific APIs.